### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by abdulmuizz0903**
Brief intro for the Repo to let people know what is it about.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# FOOD EXPIRATION MANAGER
```
Comment (lorenzovngl) [inline README.md:1]: Please change this title with the title of the project: _Food Expiration Dates_

Comment (abdulmuizz0903) [reply README.md:1]: ok no problem

**Review by lorenzovngl — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-# FOOD EXPIRATION MANAGER 
+# FOOD EXPIRATION DATES
```
[the remark this answers is not in the document]
Comment (abdulmuizz0903) [reply README.md]: Sorry, I updated it

**Review by lorenzovngl — APPROVED** [review]

**Comment by lorenzovngl** [comment]
Thank you for the contribution!

**Pull request merged**